src/API/fetch.js

```diff
@@ -0,0 +1,29 @@
+const path = 'https://www.googleapis.com/books/v1/';
```

Comment:
why foldername with uppercase?
lets rename file to bookService.js, getData move to helper.js in this folder

Reply:
Ok